index.bs

```diff
@@ -8972,6 +9018,392 @@ worker=] algorithm:
 
 </div>
 
+## The storage Module ## {#module-storage}
+
+The <dfn export for=modules>storage</dfn> module contains functionality and
+events related to storage.
+
+A <dfn>storage partition</dfn> is a namespace within which the user agent may
+organize persistent data such as [=cookies=] and local storage.
+
+A <dfn>storage partition key</dfn> uniquely identifies a [=storage partition=].
+
+### Definition ### {#module-storage-definition}
+
+[=Remote end definition=]
+
+<pre class="cddl remote-cddl">
+StorageCommand = (
+  storage.GetCookies //
+  storage.SetCookie //
+  storage.DeleteCookies
+)
+</pre>
+
+[=Local end definition=]
+
+<pre class="cddl local-cddl">
+StorageResult = (
+  storage.GetCookiesResult //
+  storage.SetCookieResult //
+  storage.DeleteCookiesResult
```

Comment:
According to [cddl spec](https://www.tech-invite.com/y85/tinv-ietf-rfc-8610.html#e-2-2-2), this should be a single `/`.
```suggestion
  storage.GetCookiesResult /
  storage.SetCookieResult /
  storage.DeleteCookiesResult
```